backends-clickhouse/src/test/scala/io/glutenproject/execution/GlutenClickHouseTPCDSParquetSuite.scala

```diff
@@ -40,6 +40,20 @@ class GlutenClickHouseTPCDSParquetSuite extends GlutenClickHouseTPCDSAbstractSui
       .set("spark.sql.files.openCostInBytes", "134217728")
   }
 
+  tpcdsAllQueries.foreach(
```

Comment:
add these tests for other suites, `GlutenClickHouseTPCDSParquetColumnarShuffleSuite`, `GlutenClickHouseTPCDSParquetColumnarShuffleAQESuite`, `GlutenClickHouseTPCDSParquetAQESuite` ?